mp4parse/src/lib.rs

```diff
@@ -3469,7 +3467,7 @@ fn read_ctts<T: Read>(src: &mut BMFFBox<T>) -> Result<CompositionOffsetBox> {
         })?;
     }
 
-    skip_box_remain(src)?;
+    check_parser_state!(src.content);
```

Comment:
I couldn't see a reason why there would ever legally be bytes remaining here and it didn't show up in any tests, but if there's a reason not to make this change, it's not necessary to fix the fuzzing issue.